docs/5-the-data-tracks/2-update-pipeline.md

```diff
@@ -10,3 +10,32 @@ push the changes (pipeline change + dvc file)
 
 
 
+1. Go back to your Jupyter Notebook and Open up `jukebox/3-prod_datascience/prod_train_save_pipeline.py` file. We will update `fetch_data` step to use DVC now. 
+
+    We need to repla
```

Comment:
This one seems half-complete

Reply:
ahh I was rushing on the train :D this shouldn't have been committed :D I'll finish that one today or tomorrow:)